src/Projects.tsx

```diff
@@ -67,6 +66,14 @@ declare interface HiddenSurprise {
 	}
 }
 
+/**
+ * Used for tracking completed project related state throughout the view/pages
+ */
+export interface CompletedProject {
+	selectedYear: number,
+	page: symbol
```

Comment:
please rename the property 'page' to 'project'

Reply:
The name convention for 'page' is used throughout the app and describes what this property is (a page symbol). There should be an issue created for the larger change here - which is to decide if 'project' describes better how these types have been used, instead of 'page'. We should gain ground on release issues before dealing with it.